api/lib/searchOperators.js

```diff
@@ -0,0 +1,121 @@
+module.exports = {
+  /**
+   * Convert ang search operators to database query
+   *
+   * @since 2.1.2
+   * @param {string} single page no, or page nos with operators
+   * @returns {string} Returns database WHERE clauses
+   * @example
+   *
+   * angToQuery: <1000+105+900-905
+   * // => {
+   *       q: 'v.PageNo < ? AND (v.PageNo BETWEEN ? AND ? OR v.PageNo IN (?))',
+   *       parameters: [ '1000', '900', '905', '105' ],
+   *       least: 0,
+   *       most: '905',
+   *       totalPages: null
+   *       }
+   */
+  angToQuery: PageNo => {
+    const hasOperatorsRegEx = /[<>+-]/gm;
```

Comment:
Was just reviewing the spec of the match func (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match) and it looks like the `m` flag doesn't exist in JS? also looks like even thought the `g` flag is in the spec, node doesn't support it?

Reply:
tested it out myself and even though that webpage says `/g` isn't supported for chrome & node, can confirm it works on the latest version of chrome & node v12.12.0